### ISSUE TITLE
AppleALC broken on Sequoia for some systems

### ISSUE THREAD

**Issue by AlfCraft07** [acidanthera/bugtracker#2415]
I installed macOS Sequoia on my AMD laptop only to notice the sound had a mute icon, I then checked Hackintool - Sound and saw the card wasn’t detected there. My sound card (Realtek ALC236) worked perfectly in Sonoma with layout 3. AFAIK, all other AMD computers in Sequoia have non-working audio with AppleALC, while it’s broken on some Intel configurations as well.

**Comment by perez987** [comment]
@AlfCraft07
I think that AMD CPUs are not supported in this bugtracker but I'm not developers member.
Did you go to the AMD OSX specific site?
[AMD OSX](https://amd-osx.com)

**Comment by AlfCraft07** [comment]
> @AlfCraft07 I think that AMD CPUs are not supported in this bugtracker but I'm not a developers member. Did you go to the AMD OSX specific site? [AMD OSX](https://amd-osx.com)

The issue, apart from AMD computers, also affects some Intel configurations according to what I saw on Reddit. Talking about AMD (just a little off-topic here), last year I updated the AMD GPU support info by forking the related Dortania guide portion, and made a PR afterwards. It's been sitting there for over a year, so please review it when you think it's appropriate.

**Comment by PSzczepanski1996** [comment]
I can confirm this issue happens on my Hackintosh too. I have `i5-13600K` build set-up on `ASUS ASUS PRIME Z790-P DDR5` motherboard. Gonna post logs and exact sound card info today.

**Comment by jazzman746** [comment]
Issue confirmed on my Asus ProArt Z490-CREATOR 10G

**Comment by rzw713-mac** [comment]
HI, if I install macOS 15 on my b450 that works perfectly on Sonoma and I use **bluetooth headphones** is ok, do I get sound out of it ?

**Comment by mikebeaton** [comment]
@PSzczepanski1996, @jazzman746 please can you provide debug logs: https://github.com/acidanthera/AppleALC/wiki/Installation-and-usage#how-to-extract-the-debug-log

**Comment by maxpicelli** [comment]
Hi all, im using VoodoHDA kext and working very fine with my all hacks Desktop Raptor Lake and Lenovo Kaby Lake 

I discovery an old script fro create .pkg installer and just works like a charm! 

"This video is for use with executable script to make easy install VoodoHDA. kext on Sequoia or any macOs. 

See this tutorial on insanelymac :  https://tinyurl.com/yc6mh3c8

Download executable script : https://tinyurl.com/tewex473  (need be log in to download) 

Try at you own risk, back-up your EFI, remove any sound kexts like AppleALC from your EFI folder. Not replace VoodooHDA, just finish install from your package created by script, gave permissions and restart you macOs system.

Its better if you can compile your own VoodooHDA.kext 

See GitHub VoodooHDA kext: https://tinyurl.com/yss87up8 "

[https://youtu.be/uJkiqg_M4Ak](url)

**Comment by Genny68** [comment]
ragazzi c'e soluzione audio per amd sequoia

**Comment by Genny68** [comment]
Guys, there is an audio solution for AMD on Sequoia

**Comment by Genny68** [comment]
puoi darmi il link

**Comment by genny1968** [comment]
tu hai risolto

**Comment by AlfCraft07** [comment]
> tu hai risolto

Sì ho risotto, ora mi funziona benissimo

**Comment by maxpicelli** [comment]
I never could imagine that people not help each other temporarily for pure nonsense. My information about VoodooHDA and patcher installer was deleted. So sad for Hackintosh users. Nevermind

**Comment by allansrc** [comment]
> I never could imagine that people not help each other temporarily for pure nonsense. My information about VoodooHDA and patcher installer was deleted. So sad for Hackintosh users. Nevermind

Can you provide that guide pls?

**Comment by genny1968** [comment]
puoi postare la guida cosi aiutiamo a tutti AlfCraft07?

**Comment by AlfCraft07** [comment]
Non c'è bisogno di nessuna guida, va solo cambiato il kext AppleALC con la versione patchata del gruppo di ChefKiss

**Comment by genny1968** [comment]
puoi darmi il link

**Comment by AlfCraft07** [comment]
Cercalo su telegram direttamente

**Comment by genny1968** [comment]
se stiamo qua a prendere in giro la gente , poverino

**Comment by maxpicelli** [comment]
[https://youtu.be/9jYfob2MgnM](url)

https://www.insanelymac.com/forum/topic/314406-voodoohda-302/?do=findComment&comment=2823924 

Need remove all kexts related with Audio (ex AppleALC)  and not put any kext on EFI folder. Just run VoodooHDA.pkg

**Comment by genny1968** [comment]
fatto ma non va mi dispiace mi da errore

**Comment by AlfCraft07** [comment]
Ig the patched AppleALC is better. Download it from here: https://t.me/ChefKissInc

**Comment by maxpicelli** [comment]
> fatto ma non va mi dispiace mi da errore

I believe that I help you on 2 forums, and maybe you need learn a little bit more about VoodooHDA, and reinstall you system. No one that im help get error.

**Comment by genny1968** [comment]
si ma dobbiamo pagare alfcraft07

**Comment by AlfCraft07** [comment]
No non si paga, ho eseguito il comando in modo che tu possa scaricarlo

**Comment by genny1968** [comment]
si ma sono entrato ma non vedo nessuno file zip

**Comment by genny1968** [comment]
scusa mi dice che non sono accessibile

**Comment by genny1968** [comment]
installato non va

**Comment by allansrc** [comment]
> fatto ma non va mi dispiace mi da errore

thanks :)

**Comment by AlfCraft07** [comment]
> installato non va

Che PC hai

**Comment by genny1968** [comment]
amd

**Comment by Andrey1970AppleLife** [comment]
It is only AppleALC bugtracker. Other alternative methods here don't discuss since it is flood. 
To write only in English.

**Comment by vit9696** [comment]
Most likely you need to change the controller patch the same way it is done for 700-series and 500-series.
See https://github.com/acidanthera/AppleALC/commit/0c82e16dd8f368278940cb1a44cc253d4ea50728

**Comment by genny1968** [comment]
> Most likely you need to change the controller patch the same way it is done for 700-series and 500-series. See [acidanthera/AppleALC@0c82e16](https://github.com/acidanthera/AppleALC/commit/0c82e16dd8f368278940cb1a44cc253d4ea50728)do you have a guide where I should go to change this value?

**Comment by allansrc** [comment]
> Most likely you need to change the controller patch the same way it is done for 700-series and 500-series. See [acidanthera/AppleALC@0c82e16](https://github.com/acidanthera/AppleALC/commit/0c82e16dd8f368278940cb1a44cc253d4ea50728)

Humm, so on data should I put the data parsed for my layout?

**Comment by PSzczepanski1996** [comment]
@mikebeaton here you go:

[Lilu_1.6.8_24.0.txt](https://github.com/user-attachments/files/17194080/Lilu_1.6.8_24.0.txt)

(Sorry for very late response, I was really busy)

**Comment by Genny68** [comment]
> @mikebeaton here you go:
> 
> [Lilu_1.6.8_24.0.txt](https://github.com/user-attachments/files/17194080/Lilu_1.6.8_24.0.txt)
> 
> (Sorry for very late response, I was really busy)

Where do I insert it?

**Comment by PSzczepanski1996** [comment]
> > @mikebeaton here you go:
> > [Lilu_1.6.8_24.0.txt](https://github.com/user-attachments/files/17194080/Lilu_1.6.8_24.0.txt)
> > (Sorry for very late response, I was really busy)
> 
> Where do I insert it?

Oh my god, this is log file, nor the patch or fix for the current issue. There is no official fix for now.

**Comment by genny1968** [comment]
ok

**Comment by PSzczepanski1996** [comment]
Can confirm that it happens on my Thinkpad T14 G1 with AMD r5 4650u build too.

**Comment by allansrc** [comment]
have anyone tested the latest build with OC 1.0.2?

**Comment by PSzczepanski1996** [comment]
Yeah, I did. And it works just fine. For some reason even before update it started working fine at my build, but not sure why (maybe I updated single kext before, and after all whole OpenCore setup). Of course I'm saying about `Z790` motherboards.

If you want to ask me about ryzen laptops, I'm pretty done with ryzentosh stuff, so I am not able to tell you if it's working or not.

**Comment by antlr72** [comment]
I have some problem with motherboard Asus H81M Plus, CPU Core i5, sound chip ALC887, with Sonoma all right but with Sequoia doesn't work

**Comment by eeshankeni** [comment]
Same here, ALC887 with alcid 13 used to work in sonoma. Seems to be broken after updated to sonoma, I'm on the latest kext. Added lilubetaall but that did not help. Im on i9-12900k

**Comment by AlfCraft07** [comment]
The issue seems to be fixed on AMD, however as for Intel I think that's a different issue that still needs fixing.
